### DISCUSSION TITLE
How to run on iOS Simulator?

### DISCUSSION THREAD

**Discussion by elsadeny** [eycorsican/leaf#32] [Q&A]
Hello, How can I run build libleaf to run support iOS simulator, just for me to test design quickly

**Comment by eycorsican** [comment]
Simply add the missing target `x86_64-apple-ios`:

https://github.com/eycorsican/leaf/blob/fb5d6799bd462ec1f905e790b60657f44c205a8b/.github/workflows/release.yml#L137

https://github.com/eycorsican/leaf/blob/fb5d6799bd462ec1f905e790b60657f44c205a8b/Makefile#L2
